pkg/reconciler/inmemorychannel/controller/inmemorychannel.go

```diff
@@ -52,6 +56,8 @@ const (
 	dispatcherRoleBindingCreated    = "DispatcherRoleBindingCreated"
 	dispatcherDeploymentCreated     = "DispatcherDeploymentCreated"
 	dispatcherServiceCreated        = "DispatcherServiceCreated"
+	secretName                      = "imc-dispatcher-tls"
+	secretKey                       = "ca.crt"
```

Comment:
I'd love to see more meaningful names here, something like `dispatcherTLSSecretName` and `caCertsSecretKey`, but we can do that as a follow up patch

Reply:
@pierDipi Updated, can you please take a look at it